goodies/bash/_zapier

```diff
@@ -31,7 +31,7 @@ _zapier() {
                 COMPREPLY=( $( compgen -W "--dest --entry" -- $cur ) )
                ;;
             test)
-                COMPREPLY=( $( compgen -W "--debug --timeout" -- $cur ) )
+                COMPREPLY=( $( compgen -W "--debug --timeout --skip-validate" -- $cur ) )
```

Comment:
It's opt-in, meaning it won't skip validation by default, but you can make it skip it by passing `--skip-validate`.